js/social-comments.js

```diff
@@ -1,23 +1,91 @@
-const socialComments = document.querySelector('.social__comments');
+const blockBigPicture = document.querySelector('.big-picture'); //модальное окно с фото и информацией о нем
+const socialCommentsCount = blockBigPicture.querySelector('.social__comment-count'); //сообщение о количестве комментариев
+const commentsCount = socialCommentsCount.querySelector('.comments-count'); //число комментариев к фото
+const printedCommets = socialCommentsCount.querySelector('.printed-comments'); //отображение в разметке числа напечатанных комментариев
 
-const commentTemplate = document.querySelector('#comment').content; //шаблон комментария
+const socialComments = document.querySelector('.social__comments'); //список комментариев к фото
+const inputComment = blockBigPicture.querySelector('.social__footer-text'); //поле ввода комментария
+const buttonDownloadComments = blockBigPicture.querySelector('.social__comments-loader'); //кнопка загрузки новых комментариев
 
+const commentTemplate = document.querySelector('#comment').content; //шаблон комментария
 const commentTemplateListElement = commentTemplate.querySelector('.social__comment'); //элемент списка комментариев, потомок шаблона
+const fragment = document.createDocumentFragment();
+
+const COUNT_COMMENTS_TO_DOWNLOAD = 5;
```

Comment:
```suggestion
const COUNT_COMMENTS_TO_SHOW = 5;
```

это количество коментов для показа